contracts/modules/Payments.sol

```diff
@@ -137,4 +138,37 @@ abstract contract Payments is RouterImmutables {
             }
         }
     }
+
+    /// @notice Wrap an amount of stETH into wstETH
+    /// @param recipient The recipient of the stETH
+    /// @param amount The amount of wstETH desired
+    function wrapSTETH(address recipient, uint256 amount) internal {
+        if (amount == Constants.CONTRACT_BALANCE) {
+            amount = STETH.balanceOf(address(this));
+        } else if (amount > STETH.balanceOf(address(this))) {
+            revert InsufficientSTETH();
+        }
+        if (amount > 0) {
+            WSTETH.wrap(amount);
+            if (recipient != address(this)) {
+                WSTETH.transfer(recipient, amount);
+            }
+        }
+    }
+
+    /// @notice Unwraps all of the contract's wstETH into stETH
+    /// @param recipient The recipient of the stETH
+    /// @param amountMinimum The minimum amount of ETH desired
+    function unwrapSTETH(address recipient, uint256 amountMinimum) internal {
+        uint256 amount = WSTETH.balanceOf(address(this));
+        if (amount < amountMinimum) {
+            revert InsufficientSTETH();
+        }
+        if (amount > 0) {
+            WSTETH.unwrap(amount);
+            if (recipient != address(this)) {
+                recipient.safeTransferETH(amount);
```

Comment:
wouldn't this be safeTransfer not safeTransferETH since STETH is erc20

Reply:
yep, will take a closer look at these functions since completing this in my late-night malaise clearly left this in a sloppy state + hopefully adding tests would catch these...